src/Microsoft.AspNetCore.SystemWebAdapters/Adapters/Features/IHttpResponseBufferingFeature.cs

```diff
@@ -16,13 +16,15 @@ namespace Microsoft.AspNetCore.SystemWebAdapters.Features;
 [Experimental(Constants.ExperimentalFeatures.DiagnosticId)]
 public interface IHttpResponseBufferingFeature
 {
-    void EnableBuffering(int memoryThreshold, long? bufferLimit);
+    void EnableBuffering(int? memoryThreshold = default, long? bufferLimit = default);
 
     ValueTask FlushAsync();
 
     [AllowNull]
     Stream Filter { get; set; }
 
+    void DisableBuffering();
```

Comment:
Adding this member to the interface is of course also a breaking change. Do we expect this to cause any issues?

Reply:
Same as above:  this interface is intentionally marked as "Experimental" so we can break it as much as we need